test/rust_src/src/dired-tests.el

```diff
@@ -35,3 +35,14 @@
         (should (= (length (system-users)) 1)))
     (progn
       (should (>= (length (system-users)) 1)))))
+
+(ert-deftest test-system-groups ()
+  (should-error (eval '(system-groups 'rms)) :type 'wrong-number-of-arguments)
```

Comment:
No need for `eval` here. `(should-error (system-groups 'rms) ...)` should work fine.